src/Core/src/Platform/Android/MauiHybridWebViewClient.cs

```diff
@@ -147,5 +148,116 @@ internal void Disconnect()
 		{
 			_handler.SetTarget(null);
 		}
+
+		private class DotNetInvokeAsyncStream : Stream
```

Comment:
I probably over engineered this, but I am using pipes to allow myself to read and write. I probably could simplify this and just block, but this appears to work.